index.html

```diff
@@ -159,7 +159,7 @@ <h2>LET’S CONNECT</h2>
             <li class="skill-item">
               <h3 class="skill__heading">
                 <span class="lang">Langguages</span>
```

Comment:
- [ ] [OPTIONAL] Please check the orthography to show a professional website 😉 (Although I understand you due to there are some orthography issues in the Figma designs 😭)